src/app/reformat/reformat.ml

```diff
@@ -19,7 +19,8 @@ let dirs_trustlist =
   ; "snarky"
   ; "_opam"
   ; "opam_switches"
-  ; ".direnv" ]
+  ; ".direnv"
```

Comment:
Is it expected? The PR is about "dune" files only.

Reply:
Yes, it is expected. It's one of two OCaml files modified in the PR. Will update the PR description.

Reply:
See https://github.com/MinaProtocol/mina/pull/16706#discussion_r1992184479